composer.json

```diff
@@ -22,7 +22,6 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "extra": {
-        "enable-patching": true,
```

Comment:
Optional: maybe preserve this line. It shouldn't hurt anything, and might prevent some confusion if someone later adds a patch to this template and doesn't remember to set this.